storybookjs/storybook
# fix: set default theme according to preferred color scheme

Issue:
Currently, the theming documentation is not completely correct because "Unless you've set your preferred color scheme as dark" is not working.  
ref #6088

## What I did
Updated the `theme` under the `defaultState` to reuse the `create` function, which, when called without arguments, already returns the `ThemeVars` according to the preferred color scheme.

## How to test

Open up a browser which supports the preferred color scheme preference, with any Storybook without a custom theme and, check that its theme is according to your preference.
